### DISCUSSION TITLE
I need to update my 0.15.0 what can I do?

### DISCUSSION THREAD

**Discussion by miguelgargallo** [pocketbase/pocketbase#3415] [Q&A] [answered]
I need to update my 0.15.0 what can I do?

**Comment by ganigeorgiev** [comment]
There are no major breaking changes to the web APIs in the recent versions, but just in case make sure to have a look at the [CHANGELOG](https://github.com/pocketbase/pocketbase/blob/master/CHANGELOG.md).

In general, you can try to backup your `pb_data` directory, download the latest executable and try to start the application with it.
If direct update to the latest v0.18.x version fails, try with a gradual update - first with v0.16.x, v0.17.x and finally v0.18.x.

**Comment by miguelgargallo** [comment]
Okay, I'm on a docker so, what are the best practices here?

**Reply by ganigeorgiev** [reply, marked as answer]
It should be the similar.
Without a sample of your Dockerfile I'm not sure how to help, but if you use a Dockerfile similar as per the one in the [docs](https://pocketbase.io/docs/going-to-production/#using-docker) then you need to bump the `PB_VERSION` arg (_make sure to have a backup of your `pb_data` volume before deploying the change_).

**Comment by miguelgargallo** [comment]
F* thanks you saved me! OMG 
![image](https://github.com/pocketbase/pocketbase/assets/5947268/bbab7bfe-48db-4101-b99d-81b6dbfaf593)
